source/support/support-overview.rst

```diff
@@ -0,0 +1,179 @@
+=======
+Support
+=======
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO users have two options for support.
+
+#. Community support from the `public Slack channel <https://minio.slack.com>`_.
+#. A paid commercial license grants access to the MinIO Subscription Network, `SUBNET <https://subnet.min.io>`_.
+   
+   For current licensing levels and pricing, refer to the |SUBNET| page.
+
+
+.. _minio-docs-subnet:
+
+SUBNET
+------
+
+SUBNET provides a support model that delivers 24/7/365 direct-to-engineer support thorugh a MinIO-builot portal that blends the best of Slack and Zendesk.
+
+Features of SUBNET include:
+
+- Security and architecture reviews
+- Access to Panic Button (depending on SLA)
+- Secure communication channel to exchange logs and software binaries
+- Unlimited seats for your team
+- Unlimited issues
+
+
+Registering for SUBNET
```

Comment:
Register your MinIO Deployment with SUBNET.

Otherwise this might read as how to actually sign up for SUBNET